Download/tools.mixer

```diff
@@ -8,9 +8,11 @@ and full data libraries takes about 1.2 GB of disk space and (except
 for Windows or rsync installations) will require a working C
 development environment on your system. This should include a C
 compiler (preferably gcc 4.2 or higher), make, binary utilities such
-as a linker, and development libraries.  You will also need the m4
-macro processor to be able to build <mixer var="GAP"/> with GMP
-support for faster large integer arithmetic; most, but not all standard
+as a linker, and development libraries.  We recommend that you also
+install a C++ compiler, as various GAP packages require one. The
+GAP build system will enable special support for such packages
```

Comment:
It is recommended to use `<mixer var="GAP"/>` instead of `GAP`, even though currently there is no visible difference how is is rendered (but if it will be, like `&GAP;` in GAPDoc, then we will not have to change anything).